web/src/table/MfaAccountTable.js

```diff
@@ -159,16 +159,20 @@ class MfaAccountTable extends React.Component {
         title={() => (
           <div>
             {this.props.title}&nbsp;&nbsp;&nbsp;&nbsp;
-            <Button style={{marginRight: "5px"}} type="primary" size="small" onClick={() => this.addRow(table)}>{i18next.t("general:Add")}</Button>
-            <Popover trigger="focus" content={
-              <QRCode
-                value={this.state.qrUrl}
-                icon={this.state.icon}
-                bordered={false}
-              />
-            }>
-              <Button style={{marginLeft: "5px"}} type="primary" size="small">{i18next.t("general:QR Code")}</Button>
-            </Popover>
+            <Button style={{marginRight: "10px"}} type="primary" size="small" onClick={() => this.addRow(table)}>{i18next.t("general:Add")}</Button>
+            {this.state.qrUrl && this.state.qrUrl.length > 0 && this.state.qrUrl.length <= 2000 ? (
```

Comment:
Directly use this.props.qrUrl

And this cannot be empty

You should check the access token error indeed based on `qrUrl={`casdoor-app://login?serverUrl=${window.location.origin}&accessToken=${this.props.account.accessToken}`}`